plugins/MidiImport/MidiImport.cpp

```diff
@@ -323,19 +333,12 @@ bool MidiImport::readSMF( TrackContainer* tc )
 	for( int s = 0; s < timeSigs->length(); ++s )
 	{
 		Alg_time_sig timeSig = (*timeSigs)[s];
-		// Initial timeSig, set song-default value
-		if(/* timeSig.beat == 0*/ true )
-		{
-			// TODO set song-global default value
-			printf("Another timesig at %f\n", timeSig.beat);
-			timeSigNumeratorPat->putValue( timeSig.beat*ticksPerBeat, timeSig.num );
-			timeSigDenominatorPat->putValue( timeSig.beat*ticksPerBeat, timeSig.den );
-		}
-		else
-		{
-		}
-
+		timeSigNumeratorPat->putValue(timeSig.beat*ticksPerBeat, timeSig.num);
+		timeSigDenominatorPat->putValue(timeSig.beat*ticksPerBeat, timeSig.den);
```

Comment:
From https://github.com/LMMS/lmms/wiki/Coding-Conventions#infix-operators:
> Spaces MUST be before and after infix operators (`=`, `+`, `-`, `*`, `/`, etc.).

Reply:
Oops! I removed the spaces around `()` but forgot that. 😄